lib/pa_ess/logger.ex

```diff
@@ -24,8 +24,6 @@ defmodule PaEss.Logger do
       "#{start_secs}"
     ]
 
-    File.mkdir("log")
-    File.write!("log/pa_ess_updates.log", line ++ ["\n"], [:append])
```

Comment:
I don't think we'll need realtime signs to log to a local file. This would just log the updates to a file when running locally to help with debugging, but I found that it's not super useful.